### DISCUSSION TITLE
Akash x Shade Protocol Owned Liquidity Discussion

### DISCUSSION THREAD

**Discussion by RemyfromPixarsRatatouille** [akash-network/community#543] [Governance Proposals]
Hello All. I hope everyone is doing well.

I am creating this post to continue a discussion around the potential creation of a Protocol Owned Liquidity Pool between Akash and Shade Protocol. Previously this had been discussed on the Discord with Hewi and in person lightly with Tyler and Adam at EthDenver 2024, but this looks to be the best stop to open up the discussion to everyone.

The Idea:

The Creation of a AKT x SHD pool on Shade Protocol using a portion of each projects community fund. This allows for the maintenance of liquidity and listing with:

- No listing cost
- No emissions from Akash needed
- No OTC between DAOs

The Benefits for Akash and Its Users:

Aside from those benefits above this also is a great expansion opportunity for more decentralized liquidity for AKT. The vast majority of liquidity and volume are on CEXs and it is listed on just a few DEXs so each pool and liquidity added to a Dex helps decentralize the token. It also gives more readily available access to the token to Secret network builders, many of whom use Akash, or are looking to expand to Akash in the near future.

The primary benefit that this pool would provide to Akash and its users revolves around something that is unique to Shadeswap. With this pool Akash users can acquire AKT privately for their deployments and product usage (making their decentralized orchestration that much more powerful). Shade utilizes the privacy aspects of Secret Network and its confidential computing, which gives all its smart contracts, and therefor any users, privacy protecting transactions and positions on Secret Network! We at Shade believe this is the right step towards a future of more decentralization and truly trustless systems, and we think it could be a great additive to Akash's users who would like to protect their personal privacy.

The Details:

Shade is open to offering up to $500k worth of SHD in order to create a $1M pool, and would suggest a minimum of $50k for the creation of a $100k pool but this is up for discussion.
-Shade is open to doing a multisig or another method as proposed by Akash community members to maintain this liquidity safely
-Each token offered up is owned entirely by its project of origin. In the event of a split all AKT would be returned to Akash community fund and vice versa for Shade
-Should one project request this LP to be ended then it is expected that within 3 weeks of that time the LP will be split and tokens returned to their respective project funds
-(For context as the poster) My name is Jeremy Schipper. I've worked with Shade on Operations and Finance (anything people/paperwork/finance related I do) for about 3 years now. I enjoy trying out new Cosmos projects, OSRS, and deathcore music. I primarily have spent my time in Secret Network but lurk in many many cosmos channels and projects. I can be reached at @JeremySchipper on Telegram as well. 
-Shade Protocol is a L2 on top of Secret Network building Defi with privacy for its users. Shade has existed since 2021 and currently has an active DEX (Shadeswap) / Stablecoin (Silk) / Derivative (stkd-scrt) and several utility based tools and dapps for the function of the dao and users on Shade (Bonds/Staking/Bridge/Etc). Shade is currently working on bringing to testnet a Liquidity Orderbook and Money Market Dapp to expand the Defi capabilities it has.
https://defillama.com/protocol/shade-protocol#information
https://app.shadeprotocol.io/swap/pools
https://twitter.com/Shade_Protocol

Active Examples of POL:
MANTA / FINA / PICA / ANDR / KUJI / STRIDE / LEVANA / WHALE / ARCH / ORAI / XPRT
Some Links to review these
https://twitter.com/AndromedaProt/status/1737851867059917023
https://fin.kujira.network/trade/kujira14u8ynhj7d8h2379yqj5q2ma0xxpvfjlrpauhzxgnj40ejq953mlsugux8d?q=all&s=shd
https://blue.kujira.network/govern/537
https://commonwealth.im/stride/discussion/13915-shd-strd-protocol-liquidity-match-proposal
https://www.mintscan.io/persistence/proposals/78

With more coming soon

Thank you all for any input and questions. Really looking forward to the talks on this!

**Comment by hewi333** [comment]
Hi @RemyfromPixarsRatatouille thank you for putting this idea in discussions for the wider audience to view. As I had mentioned in discord, we have yet to do a POL swap / use any community funds for liquidity at this time. That being said, I do think Shade is in a more unique position than just adding liquidity to another DEX. 

> The primary benefit that this pool would provide to Akash and its users revolves around something that is unique to Shadeswap. With this pool Akash users can acquire AKT privately for their deployments and product usage (making their decentralized orchestration that much more powerful).

I think the most compelling aspect is that users can acquire AKT privately and then use for deployments, something that difficult to do today. If the community feels like this would be a good use of funds to expand AKT's capabilities, I would suggest starting with a small amount and then growing out from there.

**Comment by Zblocker64** [comment]
Hey there @RemyfromPixarsRatatouille Thanks for your post. I would be more than ok with doing a trial run with the limited amount you suggested of 50k. This would allow a good test and if this proves that the liquidity is there, I wouldn't be opposed to doing more in the future. I think having a place where users can buy AKT privately would be a good add. I would vote yes if this came on chain asking for 50k.

**Comment by dominikusbrian** [comment]
Hi @RemyfromPixarsRatatouille , this is an exiting idea to work with.  I want to contribute my thought and concern from the perspective of provider and also user. In particular I want to comment on what's discussed by @heyhewi above.
Checking on https://app.shadeprotocol.io/swap/pools
SHD-TIA cost 0% in extra fee for user to gain access to this LP,
while SHD-MANTA cost 0.29%. Though that are almost nothing compared to Crypto CEX and  web2/fintech solution, that left a question for the partnership in how this is determined and how the dynamics gonna go. 

That being said, It would be interesting also how the cost-interest will play for longer term liquidity contract, like borrow and repaying AKT , so user can get AKT loan and the like. One more thing is to understand how this Shade is compared to DePIN specific DEX LP player like Loxodrome. From the way I understanding Loxodrome ve(3,3) mechanism, Akash could actually earn decent profits for the contributed liquidity. But one sure thing, most likely having several DEX will always be better on the Akash and AKT end. With that in mind, it will then simply a matter of finding the balance that works nicely overtime.

**Reply by RemyfromPixarsRatatouille** [reply]
Thanks again for the feedback!

The way Shade Protocol sets up fees with CPMM pools is if SHD LP incentives are given to a pool, the taker fee is set to 100% to the ShadeDAO & LPs purely receive incentives. This fee structure will be deprecated with the implementation of our Trader Joe style liquidity orderbook (currently on testnet) which will make market making much more competitive, profitable, and concentrated (improving the overall trading experience). This liquidity orderbook implementation is still quite far out as we will need to walk through our public testing + audit stage.
Were this SHD/AKT liquidity matching pool were to be approved, 0.29% of trading fees will be returned to liquidity providers (in this case ShadeDAO teaming up with Akash)  with a 0.01% taker fee back to the ShadeDAO.

**Comment by brewsterdrinkwater** [comment]
Jeremy talked about discussion 543 during the Akash Steering Committee meeting on June 27th, 2024. Jeremy also talked about the discussion during previous sig-economics meetings.

The video can be found here, starting at the 20 minute mark: [June 27th, 2024 Akash Network Steering Committee Monthly Meeting Video Recording](https://upv27tytyope2fseqvgnnre2au6nmmo65bs2c2fvuxtohsr5bpjq.arweave.net/o-uvzxPDnk0WRIVM1sSaBTzWMd7oZaFotaXm48o9C9M)

You can catch up on all past Akash Steering Committee meetings [here](https://github.com/akash-network/community/tree/main/committee-steering).
